docs/01_introduction/hardware.md

```diff
@@ -0,0 +1,112 @@
+# Choosing the Optimal Computer
+
+Though Python is runnable on most of modern operating systems (OS) including Windows, MacOS and Linux, it is beneficial to keep scripting under *nix environment. Here we provide a guide for beginners to choose your computing hardwares.
```

Comment:
"it is beneficial to keep scripting under *nix environment"

Can you explain why? I work all day on Windows, for many years, and I see no issue with that.